examples/connectivity/plot_canica_resting_state.py

```diff
@@ -26,6 +26,10 @@
 adhd_dataset = datasets.fetch_adhd()
 func_filenames = adhd_dataset.func  # list of 4D nifti files for each subject
 
+# print basic information on the dataset
+print('First functional nifti image (4D) is at: %s' %
+      adhd_dataset.func[0])  # 4D data
```

Comment:
I don't think it should say first here since the list is of size 1, maybe something like 'Functional 4D image is at: '